Library/Formula/mahout.rb

```diff
@@ -37,4 +37,15 @@ def caveats; <<-EOS.undent
       export JAVA_HOME=$(/usr/libexec/java_home)
     EOS
   end
+
+  test do
+    ENV["JAVA_HOME"] = `/usr/libexec/java_home`.chomp
```

Comment:
We can set JAVA_HOME for users in install:
`bin.env_script_all_files(libexec/"bin", Language::Java.java_home_env)`